bio/picard/markduplicates/environment.yaml

```
@@ -3,6 +3,6 @@ channels:
   - bioconda
   - nodefaults
 dependencies:
-  - picard =2.27.4
+  - picard =3.0.0
   - samtools =1.16.1
   - snakemake-wrapper-utils =0.5.2
```

Just out now, so we avoid a further auto-bump in the future:
```suggestion
  - snakemake-wrapper-utils =0.5.3
```